src/components/Report/Report.tsx

```diff
@@ -45,15 +46,15 @@ const Report = () => {
           categoryMainNumber={
             userReport[data].monthly_principal && `$${userReport[data].monthly_principal}`
             || userReport[data].zipcode
-            || userReport[data].downpayment_percent_saved
+            || userReport[data].downpayment_percent_saved && `%${userReport[data].downpayment_percent_saved}`
```

Comment:
is this the same as saying if this `userReport[data].downpayment_percent_saved` exits then do this `%${userReport[data].downpayment_percent_saved}`????